README.md

```diff
@@ -22,13 +22,17 @@ The RecordLinker is a service that links records from two datasets based on a se
 Set up a Python virtual environment and install the required development dependencies:
 NOTE: Sourcing the script is recommended over simply executing the script. This allows
 the virtual environment to stay active in your shell.
+
 ```bash
 source scripts/bootstrap.sh
 ```
 
+Note: If you are running WSL on a Windows machine, you may need to comment out line 8 in `bootstrap.sh` or configure your clone of the RecordLinker repo in the Ubuntu directory of your machine before running.
```

Comment:
@m-goggins thanks for adding this information.  Out of curiosity, what was the error?  Line 8 in the bootstrap.sh script is used in many of the scripts to make sure the user is in the root directory before executing the logic.  Is the `/` causing issues, or maybe `dirname`?  Wondering if we could just make a small tweak to the scripts to make them more reliable on WSL.  For instance, would this fix it?

```shell
cd "$(dirname "$(realpath "$0")")/.." || exit 1
```

Reply:
The original error was `realpath: invalid option -- 'b'` and then as a result `-bash: .venv/bin/activate: No such file or directory` because line 8 bumped me to my `Repo` folder and not `Repo/RecordLinker` where I believe it was intended to go. The same issue persists with the code you suggested.

I wonder if this code might work to ensure a user is always at the top level of the directory for the repo?

`cd "$(git rev-parse --show-toplevel)" || exit 1`

Reply:
Updated docs based on conversation with Eric yesterday. I was also able to get the local server running so I should be all set to start working with the code next week!